{
	"name": "@offirmo-private/normalize-error",
	"description": "normalize a thrown thingy, ensuring it's a proper error instance",
	"version": "0.0.1",
	"author": "Offirmo <offirmo.net@gmail.com>",
	"license": "Unlicense",
	"private": true,

	"esnext": "dist/src.es9/index.js",
	"module": "dist/src.es9/index.js",
	"main": "dist/src.es9.cjs/index.js",
	"typings": "dist/src.es9.cjs/index.d.ts",
	"source": "src/index.ts",
	"sideEffects": false,

	"peerDependencies": {
	},
	"dependencies": {
		"@offirmo-private/common-error-fields": "^0.0.1",
		"tslib": "^1"
	},

	"devDependencies": {
		"npm-run-all": "^4"
	},
	"scripts": {
		"clean": "node ../../scripts/clean.js dist",

		"build:dev:watch": "node ../../scripts/build-typescript.js --watch",
		"build:prod": "node ../../scripts/build-typescript.js",

		"dev": "run-s clean build:dev:watch",
		"build": "run-s build:prod"
	}
}
